src/main/java/de/tum/in/www1/artemis/web/rest/TextExerciseResource.java

```diff
@@ -169,7 +200,9 @@ public TextExerciseResource(TextExerciseRepository textExerciseRepository, TextE
             result.getExampleSubmissions().forEach(exampleSubmission -> exampleSubmission.setTutorParticipations(null));
         }
 
-        if (notificationText != null) {
+        // TODO: Should we leave this activated in case the exercise is changed during the exam?
```

Comment:
Honestly I'm not sure if we could reuse this notification functionality for the exam. Do students even see notifications if they are in exam mode?
I don't think Artemis supports versioning of exercises at the moment so this would also require additional work. We should discuss this with the team.

Reply:
I created a meeting page for the next meeting and added this as a discussion point.

Reply:
we do not want to support this at the moment. Exercises should not be changed during the exam!